src/FTPObject.jl

```diff
@@ -104,7 +104,9 @@ Upload IO object "file" to the FTP server and save as "file_name".
 If "file" is not specified, the file "file_name" is uploaded.
 """ ->
 function upload(ftp::FTP, file_name::AbstractString, file=nothing)
```

Comment:
There might be a need to upload something with a different name. Maybe add `upload_name::AbstractString=nothing` to each function too.